main.py

```diff
@@ -38,18 +35,35 @@ def inside_trades():
 
     buy, sell = transactions.get_trade_decisions()
     orders = helpers.place_trades(buy, sell)
-    helpers.activate_order(orders)
+    helpers.activate_orders(orders)
+
+
+def schedule_trades_for_year():
+    opening_days = helpers.get_opening_days()
+
+    for i in range(len(opening_days)):
+        scheduler.add_job(inside_trades,
+                          trigger=CronTrigger(month=opening_days[i].month,
+                                              day=opening_days[i].day,
+                                              hour=13,
+                                              minute=9,
+                                              timezone=utc))
 
 
 if __name__ == '__main__':
     scheduler = BlockingScheduler(timezone=utc)
 
-    scheduler.add_job(inside_trades,
-                      trigger=CronTrigger(day_of_week="mon-fri",
-                                          hour=10,
-                                          minute=30,
-                                          timezone=utc),
-                      name="Perform inside trades")
+    inside_trades()
+    schedule_trades_for_year()
```

Comment:
Would rather suggest you keep the scheduler as is (and remove the schedule_trades_for_year() function entirely). Then, within the inside_trades() method, you can simply place a check for whether the trading venue is currently open, with:

`if client.market_data.venues.get().results[0].is_open:
    function logic
`

Reply:
Then make sure you call inside_trades in the scheduler rather than schedule_trades_for_year

Reply:
This seems much simpler, I'll go ahead and implement this.